app/views/posts/index.html.erb

```diff
@@ -1,4 +1,6 @@
 <main class="user-posts">
+	<p><%= can? :read, @user%></p>
+	<p><%= can? :read, @posts%></p>
```

Comment:
- [ ] Hey @carlosmuerto . You are printing two boolean to the screen. I think you missed deleting them after debugging.